### ISSUE TITLE
Removal of full stops and commas in vld collection

### ISSUE THREAD

**Issue by geoffj-FUG** [FreeUKGen/FreeCENMigration#1763]
@AnneV-Learn 
Can we add a couple of lines to the revalidation script please?
I would like it to remove all full stops and commas in the place of birth and verbatim place of birth fields please and replace them with a space character. This should happen before the piece is tested.
I am spending a lot of time removing these manually as I re-validate. They are not allowed in the csv files.

Geoff

**Comment by AnneV-Learn** [comment]
A couple of lines added  ;) 
Ready for testing on Test3

**Comment by geoffj-FUG** [comment]
Anne

Works well.

I have asked for this to be deployed. I have created a new story for this code to be added to download as csvpro. It will reduce coordinators' work.

Geoff

**Comment by DeniseColbert** [comment]
Done, closing
